src/backbone.virtual-collection.js

```diff
@@ -117,6 +117,7 @@ var VirtualCollection = Backbone.VirtualCollection = Backbone.Collection.extend(
         this.trigger('change', model, this, options);
       } else {
         this._indexAdd(model);
+        this.listenTo(model, 'all', this._onAllEvent);
```

Comment:
Of course, moreover, is the collection parameter of the function '_onAdd' is really needed? (Currently it isn't for the function)